app/src/main/java/org/smartregister/addo/util/Utils.java

```diff
@@ -341,5 +342,9 @@ public static String getDayOfMonthSuffix(final int n) {
         return CoreLibrary.getInstance().context();
     }
 
+    public static String getTranslatedGender(String gender) {
```

Comment:
does this function need to be in both FamilyMemberAdapter class and here?

Reply:
@Andati - Good point, I realized I am writing the same function over and over again, so I decided to put it here. I guess I can remove it from the FamilyMemberAdapter and leave it here as a util function